### PR TITLE
Adding new issue templates

### PR DESCRIPTION
**Pull request by jtravee**
Adding new issue templates; also updating GH button on docs site.

### CODE REVIEW

File: .github/issue_template/request-an-update.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Request an Update
```
Comment (btat) [inline .github/issue_template/request-an-update.md:2]: There's a lot of overlap with the template for new features. We may want to make this one more generalized?

**Review by btat — CHANGES_REQUESTED** [review]

File: .github/issue_template/request-an-update.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+about: For fixing docs errors/typos, adding needed/missing information, updating verbiage, deleting outdated info, etc.
+title: "[Rancher Manager] "
+labels: Rancher Manager
```
[the remark this answers is not in the document]
Comment (jtravee) [reply .github/issue_template/request-an-update.md]: Corrected.

File: .github/issue_template/request-a-new-feature.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: Request a New Feature
+about: For requesting new feature(s) to be added to the docs.
+title: "[Rancher Manager] "
+labels: Rancher Manager
```
[the remark this answers is not in the document]
Comment (jtravee) [reply .github/issue_template/request-a-new-feature.md]: Corrected.

File: .github/issue_template/request-to-add-item-to-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+name: Request a Release Note
+about: For requesting items be added to the release notes.
+title: "[Rancher Manager] "
+labels: Rancher Manager
```
[the remark this answers is not in the document]
Comment (jtravee) [reply .github/issue_template/request-to-add-item-to-release-notes.md]: Corrected,

File: .github/issue_template/request-to-add-item-to-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+name: Request a Release Note
```
[the remark this answers is not in the document]
Comment (jtravee) [reply .github/issue_template/request-to-add-item-to-release-notes.md]: Removed this item.

File: .github/issue_template/request-a-new-feature.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+**Summary:**
+Briefly describe the new feature. If QA has not yet tested the new feature/process, please also file a ticket with QA for their review.
+
+
+**Details:**
+- What page(s) in the current docs does request apply to? Please include link(s).
+- What does the new process/feature do and how does it work?
+- What are the steps involved in installing/running feature? Please describe.
+- What version(s) of Rancher does the feature affect?
+- List/attach all pertinent information, e.g., screenshots, resource requirements, code blocks, workarounds, links, etc.
```
[the remark this answers is not in the document]
Comment (jtravee) [reply .github/issue_template/request-a-new-feature.md]: Rearranged info; removed the term `code blocks`.

**Comment by jtravee** [comment]
Also updated the sidebar to auto-collapse sidebar categories as discussed.

**Pull request merged**